src/store/reducers/index.js

```diff
@@ -2,12 +2,14 @@ import { combineReducers } from 'redux';
 import { routerReducer } from 'react-router-redux';
 import auth from './auth';
 import usersReducer from './users_reducer';
+import vote from './vote';
 import question from './question';
 
 const rootReducer = combineReducers({
   auth,
   usersReducer,
   question,
+  vote,
```

Comment:
unofficial convention: reducer should appear like voteReducer import from vote_reducer.js file

Purpose: easy to distinguish with another vote files.